docs/WatchPlugins.md

```diff
@@ -155,7 +155,7 @@ class MyWatchPlugin {
 
 For stability and safety reasons, only part of the global configuration keys can be updated with `updateConfigAndRun`. The current white list is as follows:
 
-- [`bail`](configuration.html#bail-boolean)
+- [`bail`](configuration.html#bail)
```

Comment:
does the link work? You can run `yarn start` in the website folder to test links etc

Reply:
Maybe I'm just doing something wrong, but I've tried `yarn start` and `yarn build` then `yarn start` in /website/ and it doesn't seem to be reflecting any of the changes from these *.md files.

Reply:
Ok, I was able to find the link in `/build` folder.  Updated to match now.

Reply:
You need to open the `next` docs, by default it renders the versioned docs. No need to build :)
You can select the version by clicking the version number in the upper left

Reply:
This document gives a 404, though, can yiu double check locally?

https://deploy-preview-7335--jest-preview.netlify.com/docs/en/next/watch-plugins

Reply:
So the link itself for `configuration#bail-number-boolean` seems to work correctly.  But the link you posted, as well as when running locally, and going to `/next/watch-plugins` both seem to return page not found.  Seems to be partially broken in master: https://jestjs.io/docs/en/next/watch-plugins   Looks like it's because of `original_id` inside the header.  I've removed it because I don't see any references to that property outside of versioned docs and removing it seems to fix that link so we can test the updated link.